baseledger/x/bridge/types/message_ubt_deposited_claim.go

```diff
@@ -67,9 +67,6 @@ func (msg *MsgUbtDepositedClaim) ValidateBasic() error {
 	if _, err := sdk.AccAddressFromBech32(msg.Creator); err != nil {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "orchestrator")
 	}
-	if _, err := sdk.AccAddressFromBech32(msg.BaseledgerReceiverAccountAddress); err != nil {
```

Comment:
i dont think the rest of these should be removed from here, same for other claim, will follow up offline with this